src/components/hand-controls.js

```diff
@@ -245,23 +245,23 @@ module.exports.Component = registerComponent('hand-controls', {
     var isTrackpadActive = this.pressedButtons['trackpad'] || this.touchedButtons['trackpad'];
     var isTriggerActive = this.pressedButtons['trigger'] || this.touchedButtons['trigger'];
     var isABXYActive = this.touchedButtons['AorX'] || this.touchedButtons['BorY'];
-    var isVive = isViveController(this.el.components['tracked-controls']);
 
     // Works well with Oculus Touch and Windows Motion Controls, but Vive needs tweaks.
-    if (isGripActive) {
-      if (isVive) {
+    if (isViveController(this.el.components['tracked-controls'])) {
```

Comment:
Is possible to keep `isVive` variable? It makes the code a bit more readable

Reply:
Yep, since it was only used once it felt redundant, but easy enough to put back in.